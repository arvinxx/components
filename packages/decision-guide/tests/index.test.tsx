import React from 'react';
import { render } from '@testing-library/react';

import DecisionGuide from '@arvinxu/decision-guide';

describe('DecisionGuide', () => {
  it('默认状态', () => {
    const { container } = render(<DecisionGuide />);
    expect(container).toMatchSnapshot();
  });
});
