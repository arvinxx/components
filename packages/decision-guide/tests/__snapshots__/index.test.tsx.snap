// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DecisionGuide 默认状态 1`] = `
<div>
  <div>
    <div
      class="ant-pro-card"
    >
      <div
        class="ant-pro-card-header"
      >
        <div
          class="ant-pro-card-title"
        >
          决策引导
        </div>
        <div
          class="ant-pro-card-extra"
        />
      </div>
      <div
        class="ant-pro-card-body"
      >
        <div
          class="ant-space ant-space-horizontal ant-space-align-center avx-guide-item"
        >
          <div
            class="ant-space-item"
            style="margin-right: 8px;"
          >
            <div>
              开始
            </div>
          </div>
          <div
            class="ant-space-item"
          >
            <span
              aria-label="right"
              class="anticon anticon-right"
              height="56"
              role="img"
              width="56"
            >
              <svg
                aria-hidden="true"
                data-icon="right"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                />
              </svg>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
